src/tools/rbd_mirror/Mirror.cc

```diff
@@ -103,6 +111,32 @@ class FlushCommand : public MirrorAdminSocketCommand {
   Mirror *mirror;
 };
 
+class SyncSetMaxCommand : public MirrorAdminSocketArgsCommand {
```

Comment:
Unneeded if you are always pulling the max sync from ```_conf``` since there are already asok commands for updating the configuration of a running process.